src/test/java/uk/gov/hmcts/reform/ccd/documentam/client/dmstore/DocumentStoreClientTest.java

```diff
@@ -323,19 +288,24 @@ void testShouldReturnEmptyWhenPatchDocumentReturnsNotFound() {
     }
 
     @Test
-    void testShouldRaiseExceptionWhenPatchDocumentFails() {
+    void testShouldReturnLeftWhenPatchDocumentFails() {
```

Comment:
The subject under test no longer throws and exception but return the failure(exception) in the left arm of the `Either`.
Perhaps it can be `testShouldReturnSuccessWhen...` and  `testShouldReturnFailureWhen...`